routers/api/v1/repo/release.go

```diff
@@ -8,8 +8,8 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/modules/setting"
-
 	api "code.gitea.io/gitea/modules/structs"
+	release_service "code.gitea.io/gitea/services/release"
```

Comment:
The underscore seems non-go like. Could this not just be "release" ?

Reply:
If all variable named `rel` but not `release` that I think use `release` OK, otherwise it will conflict.

Reply:
I renamed it to releaseservice